UPGRADE-15.0.md

```diff
@@ -814,6 +814,11 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 
 -   see #project-base-diff to update your project
 
+#### rename variable differentDeliveryAddress into isDeliveryAddressDifferentFromBilling ([#3161](https://github.com/shopsys/shopsys/pull/3161))
+
+-   bool variable differentDeliveryAddress was renamed into more suitable isDeliveryAddressDifferentFromBilling across the project
```

Comment:
maybe it would be nice to be more explicit here, something like this:

```suggestion
-   FE API: `OrderDecorator.types.yaml` and `OrderInputDecorator.types.yaml`: differentDeliveryAddress was renamed into more suitable isDeliveryAddressDifferentFromBilling
````

I would say that your original text could be used in the Storefront part of the upgrade notes